src/ProductConstructionService/ProductConstructionService.Api/Queue/JobRunners/TextJobRunner.cs

```diff
@@ -1,20 +1,18 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-
 using ProductConstructionService.Api.Queue.Jobs;
 
 namespace ProductConstructionService.Api.Queue.JobRunners;
 
 public class TextJobRunner(ILogger<TextJobRunner> logger) : IJobRunner
 {
-    ILogger<TextJobRunner> _logger = logger;
+    readonly ILogger<TextJobRunner> _logger = logger;
```

Comment:
```suggestion
    private readonly ILogger<TextJobRunner> _logger = logger;
```
not sure how I missed that